.changeset/silent-tomatoes-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@makeswift/runtime': minor
+---
+
+Add new icon support to runtime
```

Comment:
For the changesets we probably want to use a more "human" tone instead of the tone we use for commit messages. i.e., feel free to use full sentences, punctuation, etc.